maths/find_lcm.py

```diff
@@ -5,20 +5,24 @@
 
 def find_lcm(num_1, num_2):
     """Find the LCM of two numbers."""
-    max_num = num_1 if num_1 > num_2 else num_2
+    if num_1>=num_2:
+        max_num=num_1
+    else:
+        max_num = num2
+    
     lcm = max_num
-    while True:
-        if ((lcm % num_1 == 0) and (lcm % num_2 == 0)):
+    while True: #Run the loop until lcm value becomes divisible by both num_1 and num_2
+        if ((lcm % num_1 == 0) and (lcm % num_2 == 0)): 
             break
         lcm += max_num
     return lcm
 
 
 def main():
     """Use test numbers to run the find_lcm algorithm."""
-    num_1 = 12
-    num_2 = 76
-    print(find_lcm(num_1, num_2))
+    num_1 = int(input())   #get the number1 input from user 
```

Comment:
These three comment can be removed as they do not add anything that is not in the code.

Please change to __num_1 = int(input().strip())__ to deal with user input of `<space>12<space>`